frontend/src/app/projects/project-wrapper/project-wrapper.component.ts

```diff
@@ -50,7 +50,7 @@ export class ProjectWrapperComponent implements OnInit, OnDestroy {
     this.projectService.clearProject();
     this.modelService.clearModel();
     this.modelService.clearModels();
-    this.projectUserService.projectUser.next(undefined);
+    this.projectUserService._projectUser.next(undefined);
```

Comment:
Isn't there a better way, than to access this "private-by-convention" attribute?